src/client/pfs/pfs.proto

```diff
@@ -475,11 +480,16 @@ service API {
 
   // File rpcs
   // PutFile writes the specified file to pfs.
+  // TODO(kdelga): Why does the generated pb.go not require anything to do with the PFR stream.
   rpc PutFile(stream PutFileRequest) returns (google.protobuf.Empty) {}
   // CopyFile copies the contents of one file to another.
   rpc CopyFile(CopyFileRequest) returns (google.protobuf.Empty) {}
   // GetFile returns a byte stream of the contents of the file.
   rpc GetFile(GetFileRequest) returns (stream google.protobuf.BytesValue) {}
+  // GetFileStream returns a GetFileResponse stream with a non-empty file for only the first message.
+  rpc GetFileStream(GetFileRequest) returns (stream GetFileResponse) {}
```

Comment:
To match with `GetBlocks` in this file I'd call this `GetFiles` although it's also worth considering if maybe this functionality should just be folded in `GetFile`. My sense is no, since it's likely going to have an even more divergent signature at some point.

Reply:
I do think it makes sense to fold it into the GetFile, but probably best to wait until after we've reached the desired state of the signature where it both takes a stream of GetFileRequest and returns a stream of GetFileResponse.

Renamed to GetFiles for now.